sample/allofhands_hawaii2016/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Email": "dev@fossasia.org",
+  "App_Name": "FBF8",
+  "Api_Link": "https://raw.githubusercontent.com/mahikaw/open-event/hawaii/sample/allofhands_hawaii2016"
+}
```

Comment:
The link is fine as far as I know and things are loading from this link, could you please send me the logs for the crash?